ios/RNTPTDocumentView.m

```diff
@@ -3565,6 +3581,11 @@ - (void)rnt_documentViewControllerTextSearchDidStart:(PTDocumentBaseViewControll
     }
 }
 
+- (BOOL)rnt_documentViewControllerIsPresetBarEnabled:(PTDocumentBaseViewController *)documentViewController
```

Comment:
Where does this method come from? I don't see the `PTToolGroupToolbar.presetsEnabled` API being used anywhere so not sure how this works.

Reply:
changed all naming of hidePresetsBar to presetsToolbarHidden across all 3 files:
https://www.pdftron.com/api/ios/Classes/PTToolGroupToolbar.html#/c:objc(cs)PTToolGroupToolbar(py)presetsToolbarHidden